datalake/src/main/java/com/sequenceiq/datalake/service/resize/recovery/ResizeRecoveryService.java

```diff
@@ -90,6 +96,13 @@ public SdxRecoveryResponse triggerRecovery(SdxCluster sdxCluster, SdxRecoveryReq
             switch (actualStatusForSdx.getStatus()) {
                 case STOP_FAILED:
                     return new SdxRecoveryResponse(sdxReactorFlowManager.triggerSdxStartFlow(sdxCluster));
+                case STOPPED:
+                    if (sdxCluster.isDetached()) {
+                        //return new SdxRecoveryResponse(sdxReactorFlowManager.triggerSdxResizeRecovery(sdxCluster, null));
+                        throw new NotImplementedException("Cluster is currently in an unrecoverable state");
```

Comment:
Reattaching can technically be done via the recovery flow chain but there has definitely not been enough testing to ensure it covers all possible cases.

I think this is good for now, but in the future we may need to extend this to cover that too. Would be nice to have as much recovery as possible via the UI/CLI.